### DISCUSSION TITLE
How read сonvert PNG, JPG images to YCbCr?

### DISCUSSION THREAD

**Discussion by Zerpico** [SixLabors/ImageSharp#2672] [Q&A]
_Using SixLabors.ImageSharp Version=2.1.6_

Hello,
I was trying to find a way to convert files (PNG, JPEG) to YCbCr (YUV). I can't find how to do this in tests or examples. 
Now I use OpenCVSharp to read and color convertion, but I would like to rewrite everything with ImageSharp
```csharp
unsafe
{
    using var mat = Cv2.ImRead(filename, ImreadModes.Unchanged);
    using var yuv420Mat = mat.CvtColor(ColorConversionCodes.BGR2YUV_I420);
    int stride = (int)yuv420Mat.Step();
    int dataSize = stride * yuv420Mat.Height;

    Span<byte> buffer = new Span<byte>(yuv420Mat.Data.ToPointer(), dataSize);
}
```

**Comment by JimBobSquarePants** [comment]
>Using SixLabors.ImageSharp Version=2.1.6

Why?

>I was trying to find a way to convert files (PNG, JPEG) to YCbCr (YUV)

You want a buffer filled with YCbCr equivalents of the pixels? You would use the [ColorSpaceConverter](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.ColorSpaces.Conversion.ColorSpaceConverter.html)

**Comment by Zerpico** [comment]
>You want a buffer filled with YCbCr equivalents of the pixels? You would use the [ColorSpaceConverter](https://docs.sixlabors.com/api/ImageSharp/SixLabors.ImageSharp.ColorSpaces.Conversion.ColorSpaceConverter.html)

This support on v.2.1.6 ?

**Reply by JimBobSquarePants** [reply]
How about you answer my question
